apps/src/blockly/customBlocks/googleBlockly/mutators/procedureDefMutator.js

```diff
@@ -81,18 +82,25 @@ export const procedureDefMutator = {
         this.description = node.textContent;
       }
     }
+
+    this.userCreated = readBooleanAttribute(
```

Comment:
@mikeharv I don't think the `domToMutation` function needs to be updated to read the `deletable` property, because it doesn't seem like it needs to be set on the mutator/mainline Blockly is set up to read it from the block itself when `saveExtraState` reads it from `this.isDeletable()`. Does that sound right?